README.md

```diff
@@ -1,4 +1,37 @@
 # rc-spanishIdValidator [Work in progress] 
 
-This is and AngularJS directive which provides validation for common Spanish official ID numbers like DNI or NIF.
+This is and AngularJS library which provides validation directives for common Spanish official ID numbers like DNI or NIF.
 
+###Usage and examples
+
+The usage of the validation directives is very easy.
+Each input you want to validate must contain the **ng-model** and the directive, **rc-dni** in this example. Now the directive will set invalid the input if it is not correct.
```

Comment:
I agree with Braulio.
The snapshot+input, will it be better to show in the beginning of the document?
In this section, we could indicate the website link to see the starter tutorial if we are going to have it.
What do you think about?